Jaeger opentracing version mismatch

https://mvnrepository.com/artifact/io.helidon.tracing/helidon-tracing-jaeger/1.4.1 - opentracing 0.32 is supported in jaeger 0.35, but pom contains jaeger 0.34

Issue such as with kafka:
```
Caused by: java.lang.AbstractMethodError: Receiver class io.jaegertracing.internal.JaegerTracer does not define or inherit an implementation of the resolved method 'abstract io.opentracing.Scope activateSpan(io.opentracing.Span)' of interface io.opentracing.Tracer.
        at io.opentracing.util.GlobalTracer.activateSpan(GlobalTracer.java:214) ~[opentracing-util-0.32.0.jar:?]
        at io.opentracing.contrib.kafka.TracingKafkaProducer.send(TracingKafkaProducer.java:118) ~[opentracing-kafka-client-0.1.3.jar:?]
```

It's easy to workaround though, by including jaeger client 0.35.0 in the dependencies

--- a/dependencies/pom.xml
+++ b/dependencies/pom.xml
@@ -55,7 +55,7 @@
         <version.lib.hikaricp>3.4.1</version.lib.hikaricp>
         <version.lib.inject>1</version.lib.inject>
         <version.lib.jackson>2.10.0</version.lib.jackson>
-        <version.lib.jaegertracing>0.35.0</version.lib.jaegertracing>
+        <version.lib.jaegertracing>0.35.5</version.lib.jaegertracing>
         <version.lib.jakarta-persistence-api>2.2.2</version.lib.jakarta-persistence-api>
         <version.lib.jandex>2.1.1.Final</version.lib.jandex>
         <version.lib.jaxb-api>2.3.0</version.lib.jaxb-api>

--- a/dependencies/pom.xml
+++ b/dependencies/pom.xml
@@ -55,7 +55,7 @@
         <version.lib.hikaricp>3.4.1</version.lib.hikaricp>
         <version.lib.inject>1</version.lib.inject>
         <version.lib.jackson>2.10.0</version.lib.jackson>
-        <version.lib.jaegertracing>0.34.0</version.lib.jaegertracing>
+        <version.lib.jaegertracing>0.35.0</version.lib.jaegertracing>
         <version.lib.jakarta-persistence-api>2.2.2</version.lib.jakarta-persistence-api>
         <version.lib.jandex>2.1.1.Final</version.lib.jandex>
         <version.lib.jaxb-api>2.3.0</version.lib.jaxb-api>

--- a/dependencies/pom.xml
+++ b/dependencies/pom.xml
@@ -39,6 +39,7 @@
         <version.lib.activation-api>1.2.0</version.lib.activation-api>
         <version.lib.annotation-api>1.3.1</version.lib.annotation-api>
         <version.lib.apache-httpclient>4.5.10</version.lib.apache-httpclient>
+        <version.lib.apache-thrift>0.13.0</version.lib.apache-thrift>
         <version.lib.brave-opentracing>0.35.0</version.lib.brave-opentracing>
         <version.lib.cdi-api>2.0</version.lib.cdi-api>
         <version.lib.eclipselink>2.7.4</version.lib.eclipselink>
@@ -139,6 +140,11 @@
                 <groupId>io.jaegertracing</groupId>
                 <artifactId>jaeger-client</artifactId>
                 <version>${version.lib.jaegertracing}</version>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.thrift</groupId>
+                <artifactId>libthrift</artifactId>
+                <version>${version.lib.apache-thrift}</version>
             </dependency>
             <dependency>
                 <groupId>io.projectreactor</groupId>

--- a/dependencies/pom.xml
+++ b/dependencies/pom.xml
@@ -142,6 +142,7 @@
                 <version>${version.lib.jaegertracing}</version>
             </dependency>
             <dependency>
+                <!-- Enforce higher version than jaeger-client brings in -->
                 <groupId>org.apache.thrift</groupId>
                 <artifactId>libthrift</artifactId>
                 <version>${version.lib.apache-thrift}</version>

